source/code/source/playfab/PlayFabAndroidHttpPlugin.cpp

```diff
@@ -598,7 +618,8 @@ namespace PlayFab
     void PlayFabAndroidHttpPlugin::SetResponseAsBadRequest(RequestTask& requestTask)
     {
         CallRequestContainer& requestContainer = this->requestingTask->RequestContainer();
-        ProcessResponse(*(this->requestingTask), 400); // 400 Bad Request
+        std::string temp = "NoID-BadRequest";
```

Comment:
This "id not set" default be set via requestContainer.SetRequestID() if there's a failure above.

Reply:
Though, it's not in-line with the "" that every other platform uses...